src/Type/Definition/ResolveInfo.php

```diff
@@ -214,6 +216,126 @@ public function getFieldSelection(int $depth = 0): array
         return $fields;
     }
 
+    /**
+     * Helper method that returns names of all fields selected in query for
+     * $this->fieldName up to $depth levels.
+     * For each field there is an "aliases" key regrouping all aliases of this field
+     * (if a field is not aliased, its name is present here as a key)
+     * For each of those "alias" you can find "args" key containing the arguments of the alias and the "fields" key
+     * containing the subfield of this field/alias. Each of those field have the same structure as described above.
+     *
+     * Example:
+     * query MyQuery{
```

Comment:
```suggestion
```

Reply:
Should I also update the one from the function getFieldSelection?